Library/Homebrew/cask/lib/hbc/utils.rb

```diff
@@ -36,6 +36,13 @@ def odebug(title, *sput)
   puts sput unless sput.empty?
 end
 
+def highlight_installed(token)
+  if (Hbc.caskroom/token).directory?
```

Comment:
Try `Cask.new(token).installed?`, so we still have the same `#installed?` logic everywhere.